assets/js/components/dashboard-sharing/DashboardSharingSettings/index.test.js

```diff
@@ -184,40 +184,6 @@ describe( 'DashboardSharingSettings', () => {
 			).toBeInTheDocument();
 		} );
 
-		it( 'should set sharing management to `Any admin signed in with Google` and disabled if a module has shared ownership', () => {
```

Comment:
Removed as it just checks if the shared ownership modules have their dropdown disabled. This is no longer relevant because we've removed those drop-downs as a whole in this PR.